src/base/File.h

```diff
@@ -57,7 +57,7 @@ class FileOpeningConfig
 #else
     int flockMode = LOCK_UN; ///< 2nd flock(2) parameter
 #endif
-    static const unsigned int RetryGapUsec = 500000; /// pause before each lock retry
+    const unsigned int RetryGapUsec = 500000; ///< pause before each lock retry
```

Comment:
with the removal of `static` the name should now start with a lowercase character.

Reply:
Done in b17fe4d to avoid arguing. I adjusted PR description accordingly.